src/util.js

```diff
@@ -49,16 +49,16 @@ export function format_memory_and_limit(usage, limit) {
         return "";
 
     let mtext = "";
-    let units = 1000;
+    let unit;
     let parts;
     if (limit) {
-        parts = cockpit.format_bytes(limit, units, true);
+        parts = cockpit.format_bytes(limit, undefined, { separate: true });
```

Comment:
imho, this `separate` business is the ugliest part of the existing API.  No cleanup would be complete without taking an axe to this...

Reply:
I :100: agree. That's why I just sent #1673, but I realize that it may be more controversial. This PR should hopefully be straightforward, so I sent it separately.

Reply:
It is interesting how it went from `bool => object`

Reply:
Note that we have a lot of separate still left in storage as common pattern.